scipy/signal/wigner.py

```diff
@@ -0,0 +1,95 @@
+"""Tools based on the Wigner quasiprobability distributions.
+"""
+
+import numpy as np
+from scipy.fft import fft
+
+
+__all__ = ['wvd']
+
+
+def wvd(x, fs=1.0, resolution=1, win_size=None):
+    """
+    Compute the Wigner-Ville distribution.
+
+    Parameters
+    ----------
+    x : array_like
+        Time series of measurement values.
+    fs : float, optional
+        Sampling frequency of the `x` time series. Defaults to 1.0.
+    resolution : int, optional
+        Resolution, number of samples between windows. Defaults to 1.
+    win_size : int, optional
+        Window size, length of frequency axis. Defaults to len(x) // 2.
+
+    Returns
+    -------
+    t : ndarray
+        Array of time samples.
+    f : ndarray
+        Array of frequency samples.
+    wv : ndarray
+        Wigner-Ville distribution of `x`.
+
+    References
+    ----------
+    .. [1] S. Mallat, "A Wavelet Tour of Signal Processing (3rd Edition)",
+        Academic Press, 2009.
+
+    Examples
+    --------
+    >>> import matplotlib.pyplot as plt
+    >>> import numpy as np
```

Comment:
this import isn't needed, it's assumed in all our examples

Reply:
Done.